toponetx/classes/cell_complex.py

```diff
@@ -11,12 +11,14 @@
 from collections import defaultdict
 from collections.abc import Hashable, Iterable
 from itertools import pairwise, zip_longest
+from typing import Any, Dict, Optional, Set, Tuple
```

Comment:
TopoNetX is Python 3.9+, so you can use the native `dict`, `set` and `tuple` classes with subscripting. The dedicated classes in `typing` have been deprecated.

Reply:
Oh, my bad. I thought those were still up-to-date, thank you.